modules/local/jaffal.nf

```diff
@@ -0,0 +1,23 @@
+process JAFFAL {
+    tag "$meta.id"
+    echo true
+    label 'process_medium'
+
+//    conda     (params.enable_conda ? "bioconda::nanolyse=1.2.0" : null)   //need conda container
```

Comment:
No conda or singularity?

Reply:
Yes, I have added conda for now. I am contacting the developers for JAFFAL to update their biocontainer, which is not working right now